setup.py

```diff
@@ -253,7 +255,8 @@ def long_description():
 
     # Dependencies.
     install_requires=[
-        'psutil >= 5.2.2'
+        'psutil >= 5.2.2',
+        'future'
```

Comment:
What's using `future`?

Reply:
The rtt.py example uses future for "builtins", which exposes the "input" Python3 built-in down to Python2, which otherwise only has "raw_input".

Reply:
http://python-future.org/compatible_idioms.html#raw-input

Reply:
Ah, I was searching for `future` and coming up blank.  Makes sense.